com.microsoft.mrtk.input/Interactors/InteractorVisuals/RingReticle.cs

```diff
@@ -49,7 +50,7 @@ public bool FadeEnabled
 
         private float previousNearFadeValue;
 
-        void Awake()
```

Comment:
Is there a reason for this?

Reply:
Done to match other visual components such as the line visuals. I don't think the performance impact should impact things much and it should keep behavior slightly more consistent.

Reply:
It looks like this might stomp on some values that seemed to be intended to be cached from the script's original settings, like `previousNearFadeValue`.

Now, if I start my app, disable the reticle, and reenable it, `previousNearFadeValue` will be overwritten with whatever the value happened to be at the time I disabled it instead of the true "original" value. Is that intended?

Reply:
I think you're right. I'll move this material caching back into `Awake()`, but I don't think there's any mechanism at the moment to set a cursor to be non-fading, so this scenario is a bit over engineered at the moment.

Reply:
Correction, it's able to be done, but only from an inspector setting, and settings are only applied when the reticle is disabled and then enabled again